### DISCUSSION TITLE
Does Fast-Stable-Diffusion automatically download the latest SD version for us?

### DISCUSSION THREAD

**Discussion by edwardk1234** [TheLastBen/fast-stable-diffusion#1144] [Q&A]
If it does not download latest version, how do I set it to download the latest version?

Thanks!

**Comment by TheLastBen** [comment]
it always updates the A1111 repo

**Comment by Topzie** [comment]
Could you make it optional? I rather use an older working version until I'm sure there are no issues with the update.

Wish you a Merry Christmas and thank you very much for all the great stuff you're doing here. 🎄❤️💚
